corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -179,6 +179,7 @@ hqDefine("cloudcare/js/formplayer/menus/api", function () {
     FormplayerFrontend.getChannel().reply("entity:get:details", function (options, isPersistent) {
         options.isPersistent = isPersistent;
         options.preview = FormplayerFrontend.currentUser.displayOptions.singleAppMode;
+        options.selectedValues = sessionStorage.selectedValues;
```

Comment:
Does the `get_details` request use selected values?

Reply:
right, I was getting ahead of myself